cpp/src/arrow/util/parsing.h

```diff
@@ -24,6 +24,8 @@
 #include <string>
 #include <type_traits>
 
+#include <double-conversion/double-conversion.h>
```

Comment:
`arrow/util/parsing.h` is being installed, since you are including a thirdparty header, we should stop installing the header

Reply:
That's a good point.

Reply:
Ok, the header should not be installed anymore.